docs/project/faq.md

```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
     interoperability wrappers.
 -   Test-driven verification that migrations are correct.
 
+### Where should I ask questions about Carbon Language?
+
+Please ask questions and hold discussions either by using
+[GitHub Discussions](https://github.com/carbon-language/carbon-lang/discussions)
+or the
+[Carbon Language Discord server](https://discord.com/channels/655572317891461132/996113200464543804).
```

Comment:
```suggestion
or
[#language-questions on Discord](https://discord.com/channels/655572317891461132/998959756045713438).
```

I admit I'm not sure whether we should point at the server versus the server invite, but let's use #language-questions.